tests/appsettings.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+# We need to override these env variables for test context before the
+# "settings" and "app" modules are initialized since they will read them.
+# After doing so, we re-export them so that all test files refer to this
+# initialization module instead of the original ones.
+
+os.environ["SECRET_KEY"] = "test_secret"
+os.environ[
+    "SQLALCHEMY_DATABASE_URI"
+] = "mysql+mysqlconnector://test:test@127.0.0.1/test"
+os.environ["ISPYB_AUTH"] = "tests/config/auth.yml"
```

Comment:
I wonder if we cant leverage `.env` here with a `.env.test` or something, so we dont have to define the variables in python. This would also mean they could be overridden locally if you run pytest manually

https://stackoverflow.com/a/66038916

Reply:
That may be more complicated than needed, we should be able to do this:
https://fastapi.tiangolo.com/pt/advanced/settings/#settings-and-testing